src/components/Section/index.tsx

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import type {StyleProp, ViewStyle} from 'react-native';
+import {View} from 'react-native';
+import type {ValueOf} from 'type-fest';
+import type {MenuItemWithLink} from '@components/MenuItemList';
+import MenuItemList from '@components/MenuItemList';
+import Text from '@components/Text';
+import useThemeStyles from '@hooks/useThemeStyles';
+import type ChildrenProps from '@src/types/utils/ChildrenProps';
+import type IconAsset from '@src/types/utils/IconAsset';
+import IconSection from './IconSection';
+
+const CARD_LAYOUT = {
```

Comment:
Wondering if we should rename this to SECTION_CARD_LAYOUT to make it clearer when imported